junit5/src/main/resources/META-INF/services/org.junit.jupiter.api.extension.Extension

```diff
@@ -0,0 +1 @@
+cz.xtf.junit5.extensions.ServiceLogsStreamingRunner
```

Comment:
other extensions aren't enabled by default in services, why this should be an exception?

Reply:
Thanks @simkam - This was the solution that came to my mind when I though about how to enable the feature. 
Per se, it had a requirement allowing for it to be started either via annotation (on individual classes for developer UX) or via system properties (automation use case).
I thought a Before/After-AllCallback was the best way to hook to the workflow, hence the extension.
And I thought the best way for that kind of extension to be not bound to annotations was to be enabled by default and then to check for system properties or annotations in order to eventually enable the feature.

Basically what is enabled by default is a _controller_, implemented by a JUnit extension, that enable the feature (part of the core) based on some conditions but please let me know whether you'd suggest something different for that part.

Reply:
I understand, but it is different from other extensions do here in XTF. With other extensions, users need explicitly load them in their projects, no matter whether the feature is disabled or enabled.

Reply:
Sure, I see. I'll think about the changes and hopefully come up with something soon. Thanks for your feedback.

Reply:
Extension registration removed, resolving this thread.